oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/PerfLogger.java

```diff
@@ -50,6 +50,8 @@ public final class PerfLogger {
     /** The logger to which the log statements are emitted **/
     private final Logger delegate;
 
+    private long infoLogMillis;
```

Comment:
this should not be zero by default. with a zero as default (and without ensuring that all PerfLoggers used throughout the code base call `setInfoLogMillis` with their desired value) it would log everything - which would cause a huge amount of logs. I would suggest the same value as was already used prior to this PR, namely Long.MAX_VALUE

Reply:
I set it Long.MAX_VALUE. Thanks